pkgs/development/python-modules/johnnycanencrypt/default.nix

```diff
@@ -60,6 +60,12 @@ rustPlatform.buildRustPackage rec {
     numpy
   ];
 
+  # Remove with the next release after 0.5.0. This change is required
+  # for compatibility with maturin 0.9.0.
```

Comment:
Thanks, looks good from my point of view:
- [x] tested tumpa and johnnycanencrypt